sdk/src/org.graalvm.nativeimage/src/org/graalvm/nativeimage/hosted/RuntimeReflection.java

```diff
@@ -107,6 +107,10 @@ public static void register(boolean finalIsWritable, Field... fields) {
         ImageSingletons.lookup(RuntimeReflectionSupport.class).register(finalIsWritable, false, fields);
     }
 
+    public static void register(boolean finalIsWritable, boolean allowUnsafeAccess, Field... fields) {
```

Comment:
A javadoc comment should be added here to match the other register methods.

Reply:
fixed